src/client/app/components/unit/EditUnitModalComponent.tsx

```diff
@@ -69,6 +70,45 @@ export default function EditUnitModalComponent(props: EditUnitModalComponentProp
 		setState({ ...state, [e.target.name]: Number(e.target.value) });
 	}
 
+
+
+	// Log the Redux state to the console
```

Comment:
See formatting comment in edit meter. Other comments on alert, what done when issue found and commenting apply here.

Reply:
done